### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by megan-godfrey**
Adds content to the home page, including a mini leaderboard showing the top 3 servers with the lowest emissions, 

The page also includes counts of the numbers of server entries, and unique server configurations. Unique configurations are defined as records that share the same values for all fields except uuid, user_id, tags, created_at, updated_at.

To be added in future PRs

- Button styling
- Icons representing platform on mini leaderboard
- The 'Enter your server data` button will link to a different page with instructions/ script download/ uploading JSONs 
- Other mini leaderboards, including locations and clusters with lowest emissions

image

### CODE REVIEW

**Comment by Womblue** [comment]
Rebased on `main`

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1 +1,40 @@
...
+  <div class="cl-card flex-column-centred">
+    <h3>Top most carbon efficient servers</h3>
+    <p>Ranked by estimated equivalent CO<sub>2</sub> emissions per core at full load.</p>
+    <% if @devices.empty? %>
+      <p class="big-text mt-5">No server data has been entered yet</p>
+    <% else %>
+      <div class="home-leaderboard leaderboard-wrapper mt-4">
+        <div class="leaderboard-content-wrapper">
+          <% @devices.each_with_index do |device, index| %>
+            <div class="leaderboard-item-wrapper blurred-frame rank-<%= index + 1 %>">
```
Comment (Womblue) [inline app/views/home/index.html.erb:20]: Could I ask that clicking on these device entries links you to the relevant device's own page? This is accessible at `/device/<%= device[:display_name] %>` but I don't really know enough about the frontend implementation to make it link properly :sweat_smile:

**Review by Womblue — CHANGES_REQUESTED** [review]
LGTM, one small change can be made now that the individual device pages exist. I've also pushed a small change to set the displayed device names to the new names instead of being a segment of the UUID.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (megan-godfrey) [reply app/views/home/index.html.erb]: Updated to add the links on the home page and full leaderboards :+1:

**Pull request merged**